docs/getting_started.md

```diff
@@ -1,56 +1,44 @@
-<card>
 # Getting started with the Facebook SDK for PHP
 
 Whether you're developing a website with Facebook login, creating a Facebook Canvas app or Page tab, the Facebook SDK for PHP does all the heavy lifting for you making it as easy as possible to deeply integrate into the Facebook platform.
-</card>
 
-<card>
-## Autoloading & namespaces {#psr-4}
+## Autoloading & namespaces
 
 The Facebook SDK for PHP v5 is coded in compliance with [PSR-4](http://www.php-fig.org/psr/psr-4/). This means it relies heavily on namespaces so that class files can be loaded for you automatically.
 
 It would be advantageous to familiarize yourself with the concepts of [namespacing](http://php.net/manual/en/language.namespaces.rationale.php) and [autoloading](http://php.net/manual/en/function.spl-autoload-register.php) if you are not already acquainted with them.
-</card>
 
-<card>
-## System requirements {#requirements}
+## System requirements
 
 - PHP 5.4 or greater
 - The [mbstring](http://php.net/manual/en/book.mbstring.php) extension
 - [Composer](https://getcomposer.org/) *(optional)*
-</card>
 
-<card>
-## Installing the Facebook SDK for PHP {#installation}
+## Installing the Facebook SDK for PHP
 
-There are two methods to install the Facebook SDK for PHP. The recommended installation method is by using [Composer](#install-composer). If are unable to use Composer for your project, you can still [install the SDK manually](#install-manually) by downloading the source files and including the autoloader.
-</card>
+There are two methods to install the Facebook SDK for PHP. The recommended installation method is by using [Composer](#installing-with-composer-recommended). If are unable to use Composer for your project, you can still [install the SDK manually](#manually-installing-if-you-really-have-to) by downloading the source files and including the autoloader.
 
-<card>
-## Installing with Composer (recommended) {#install-composer}
+## Installing with Composer (recommended)
 
 [Composer](https://getcomposer.org/) is the recommended way to install the Facebook SDK for PHP. Simply run the following in the root of your project.
 
-~~~
+```php
 composer require facebook/php-sdk-v4
-~~~
 
-%FB(devsite:markdown-wiki:info-card {
-  content: "The Facebook SDK starting adhering to [SemVer](http://semver.org/) with version 5. Previous to version 5, the SDK did not follow SemVer.",
-  type: 'info',
-})
+> content: "The Facebook SDK starting adhering to [SemVer](http://semver.org/) with version 5. Previous to version 5, the SDK did not follow SemVer.",
+> type: 'info',
```

Comment:
@nayafia About the convertion of Facebook cards to a blockquote, I think it will be good to make a normal Markdown blockquote for card of type `info`, and prefix other cards with their type in bold.

For example, a card of type `warning` will become `> **Warning**: The content of the warning card...` while an `info` card will just become `> The content of the info card...`

What do you think about that? :)